### PR TITLE
[#22] docker compose for service

### PR DESCRIPTION
**Pull request by NoahHsu**
## Description
- update README for docker compose run applications and dependencies
## Changes
- add env for kafka and application both in docker
- add docker compose command into README 
- delete Makefile
- make tempo host a env var
## Screenshot(Optional)
-

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,3 +66,4 @@ for now just run each project by either following way
 | Shipment Command | http://localhost:8087/swagger-ui/index.html |
 | Shipment Handler | http://localhost:8088/swagger-ui/index.html |
 | Shipment Query   | http://localhost:8089/swagger-ui/index.html |
```
Comment (janbarry168) [inline README.md:68]: It is not in this PR scope, but I think we can merge all the Swagger UI into a single one

Comment (NoahHsu) [reply README.md:68]: yeah, i open an new issue to do that. we can discuss in detail under that issue.

File: Docker/kafka-docker-compose.yml [hunk 2]
```diff
@@ -2,6 +2,7 @@ version: "3"
 services:
   zookeeper:
     image: 'bitnami/zookeeper:${ZOOKEEPER_TAG:-latest}'
+    extra_hosts: [ 'host.docker.internal:host-gateway' ]
```
[the remark this answers is not in the document]
Comment (NoahHsu) [reply Docker/kafka-docker-compose.yml:5]: seems it's not a valid setting?
image

[the remark this answers is not in the document]
Comment (janbarry168) [reply Docker/kafka-docker-compose.yml:5]: I originally mean this
image
but it seems to fail too, alright...forget it
image

**Comment by NoahHsu** [comment]
temp solution for #41

**Pull request merged**